build.gradle

```diff
@@ -291,6 +291,12 @@ dependencies {
 
   implementation group: 'org.camunda.bpm', name: 'camunda-external-task-client', version: '1.4.0'
   implementation group: 'org.camunda.bpm.extension.rest', name: 'camunda-rest-client-spring-boot-starter', version: '0.0.4'
+  implementation ('org.mybatis:mybatis') {
+    version {
+      strictly '3.5.6'
```

Comment:
we can also add under dependencyManagement a `dependencySet` to pin down the version. We don't need to explicitly add the dependency here